bin/pgenv

```diff
@@ -183,6 +183,87 @@ pgenv_check_dependencies(){
     fi
 }
 
+# This function provides the specified version from an alias name.
+# WARNING: this does not include beta versions!!!
+#
+# Accepted parameters are either 'latest' or 'oldest' to select
```

Comment:
The closer complement to "latest" is "earliest"; the closer complement to "oldest" is "newest". I think I prefer the fomer.